common/djangoapps/student/tests/test_email.py

```diff
@@ -132,6 +137,24 @@ def _assert_activation_email(self, subject, body_fragments):
         for fragment in body_fragments:
             self.assertIn(fragment, msg.body)
 
+    @mock.patch('student.tasks.log')
+    def test_send_email_to_inactive_user(self, mock_log):
+        """
+        To verify that activation email has been sent to
+        an un-activated user(logged-in via social-auth)
```

Comment:
You can be logged in as an inactive user (namely right after you register). Once you're logged out though, you can't be logged back in.